WTWSMS/common/religions/00_religions.txt

```diff
@@ -2516,12 +2558,13 @@ pagan_group = {
 		female_temple_holders = yes
 		feminist = yes
 		defensive_attrition = yes
-		max_consorts = 3
 		peace_prestige_loss = yes # Lose Prestige while at peace
 		raised_vassal_opinion_loss = no
 		allow_looting = yes
 		reformed = celtic_pagan_reformed
 		aggression = 1.5 # AI aggression factor
+		bs_marriage = yes
+		pc_marriage = yes
```

Comment:
If you believe this is still insufficient grounds to add it, though, could we maybe compromise in some way? Since there is clearly historical evidence of nuclear incest which was quasi-accepted.